areaofcircle.py

```diff
@@ -0,0 +1,12 @@
+class AreaOfCirlce:
+
+    def __init__(self):
+        self.radius = float(input("Please enter the radius value to find the area of a cirlce"))
+        print("The area of a circle is:{0} ".format(self.area1(self.radius)))
+
+    def area1(self,radius):
+        return 3.14*self.radius**2
+        # print("The radius is {0}".format(radius))
+
+
+ar = AreaOfCirlce()
```

Comment:
If you are not using the object `ar` later, create the object without assigning it to variable.